app/models/mixins/retirement_mixin.rb

```diff
@@ -253,15 +253,16 @@ def valid_zone?
   end
 
   def system_context_requester
-    if try(:evm_owner_id).present?
-      if User.find_by(:id => evm_owner_id).present?
-        return evm_owner
-      else
-        $log.info("#{name} has evm_owner_id present but no user with that id found so defaulting to admin.")
-        return User.super_admin
-      end
-    end
+    user = User.find_by(:id => evm_owner_id) if respond_to?(:evm_owner_id)
     $log.info("System context defaulting to admin user because owner of #{name} not set.")
```

Comment:
This message does not make sense.

Reply:
It's from your comment on https://github.com/ManageIQ/manageiq/pull/19319#discussion_r327119464

Reply:
And yes, it does. The object doesn't have an evm owner.